website/content/docs/templates/hcl_templates/contextual-variables.mdx

```diff
@@ -132,6 +132,8 @@ parenthesis may through off your shell escaping otherwise.
 
 # HCP Packer Iteration ID
 
+~> **Note**: Deprecation: Contextual Variable `iterationID` has been deprecated packer context. Please use `versionFingerprint` variable instead.
```

Comment:
I'm not sure we can suggest using `versionFingerprint` without encouraging them to update the data sources that use it at the same time.
I would maybe reword this here so it explains the rationale behind the change.

```suggestion
~> **Note**: the `packer.iterationID` variable is now deprecated and will be removed in a future version of Packer. HCP Packer Versions should be accessed with their fingerprint instead. The `packer.versionFingerprint` variable is now exposed to be used in its stead with the new HCP Packer data sources.
```